doc/reference/reference_lua/box_error.rst

```diff
@@ -173,6 +177,32 @@ Below is a list of all ``box.error`` functions.
           - file: '[string "e = box.error.new{code = 555, reason = ''Arbit..."]'
             line: 1
         ...
+        tarantool> box.error.last()
+        ---
+        - nil
 
+.. _box_error-set:
 
+.. function:: box.error.set(error object)
 
+    Set an error in diagnostic area explicitly. Accepts error object and set it
+    as the last system error (i.e. becomes available via ``box.error.last()``).
+
+    **Example:**
```

Comment:
I have different output here, please double-check with the dev team.

Tarantool 2.4.1-6-g83c0356a1
Ubuntu 18.04

```
tarantool> box.cfg()
tarantool> err = box.error.new({code = 111, reason = "cause"})
---
...
tarantool> tarantool> box.error.last()
---
- error: '[string "return tarantool> box.error.last()"]:1: attempt to compare two
    nil values'
...
tarantool> box.error.set(err)
---
...

tarantool> box.error.last()
---
- cause
...
```

Reply:
fixed in f614b5d